src/cpp/shared/scripting/callback.cpp

```diff
@@ -18,4 +22,14 @@ void ironfist_lua_push(std::string arg) {
 }
 
 void ironfist_lua_pushmulti() {
+}
+
+template <>
+nonstd::optional<bool> GetLuaResult(lua_State *L, int arg) {
```

Comment:
`arg` isn't used here.  Do you need it?

Reply:
Woops; I left in some hardcoded values as I was getting this to work. Fixed.